packages/analytics-nextjs/src/context.test.tsx

```diff
@@ -40,6 +42,32 @@ describe('AnalyticsContextProvider', () => {
         expect(getByText(/analytics/i)).toHaveTextContent('disabled');
     });
 
+    it('Loads Plausible integration when newsroom has it enabled', async () => {
+        getConsentCookieMock.mockReturnValue(true);
+
+        const { getByTestId } = render(
+            <AnalyticsContextProvider
+                newsroom={{ ...DEFAULT_NEWSROOM, is_plausible_enabled: true }}
+                isEnabled={false}
+            />,
+        );
+
+        await waitFor(() => expect(getByTestId('plausible-debug-enabled')).toBeInTheDocument());
+    });
+
+    it('Does not load Plausible integration when it is disabled', async () => {
+        getConsentCookieMock.mockReturnValue(true);
+
+        const { getByTestId } = render(
+            <AnalyticsContextProvider
+                newsroom={{ ...DEFAULT_NEWSROOM, is_plausible_enabled: false }}
+                isEnabled={false}
+            />,
+        );
+
+        await waitFor(() => expect(getByTestId('plausible-debug-disabled')).toBeInTheDocument());
```

Comment:
Isn't it enough to render this element only when `PlausibleProvider` is used? Then you'd just check if the element exists or not.

It was a bit confusing at first to see that you expect an element in both cases.

Reply:
Yeah, that would make more sense. Updating :)

Reply:
https://github.com/prezly/analytics/pull/76/commits/d0199642cc547cb73c6e983881536149959d65e9